src/seedu/addressbook/parser/Parser.java

```diff
@@ -222,7 +222,7 @@ private int parseArgsAsDisplayedIndex(String args) throws ParseException, Number
      * @param args full command args string
      * @return the prepared command
      */
-    private Command prepareFind(String args) {
+    private static Command prepareFind(String args) {
```

Comment:
you will have to update the parser test file as well as it uses `parseCommand` method